readme.md

```diff
@@ -4,6 +4,7 @@
 
 
 ## Contents
+- [Bitcoin Related](#btc)
```

Comment:
```suggestion
- [Bitcoin](#bitcoin)
```

Do you mind placing the index title section and section in the right place alphabetically?

Reply:
Done.